game/domain/definitions/attribute.lua

```diff
@@ -10,9 +10,16 @@ ATTR.INFLUENCE = {
   VIT = {'ANI', 'COR'}
 }
 
-function ATTR.DMG(attr, base)
-  local power = attr * base
-  return power/4, 3*power/4
+local floor = math.floor
+local function round(x, ...)
+  if x then
+    return floor(x + 0.5), round(...)
+  end
+end
```

Comment:
you do know we already have a round function in `common.math` right
you can just expand that function to be recursive as well instead of reimplementing it